tsconfig-base.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": [ "es2015" ],
+    "target": "es6",
```

Comment:
This is required since `@grpc/grpc-js` is distributed in ES6 and typescript fails to call the constructor of an ES6 class if you target ES5. I did verify with the primary user of daml-js that requiring es6 is not an issue.